.travis.yml

```diff
@@ -11,3 +11,12 @@ script:
   - yarn build
   - yarn test
   - yarn test:bench
+
+deploy:
+  provider: script
+  skip_cleanup: true
+  script:
+    - echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" > .npmrc
```

Comment:
`- npm set //registry.npmjs.org/:_authToken ${NPM_TOKEN}` I learned today from the `chameleon` project